docs/authoring/working_with_teams.md

```diff
@@ -0,0 +1,514 @@
+
+:::tip A sample persona for an eviction defense tool
+
+**Persona: Maria Rodriguez**
+* Age: 35
+* Location: Boston, Massachusetts
+* Occupation: Retail Worker
+* Family Status: Single mother with two children, ages 8 and 5
+* Income Level: Low-income bracket
+* Housing Status: Renting a two-bedroom apartment in a building primarily occupied by similar low-income families
+* Technology Use: Maria owns an older Android smartphone but has limited data. She's used to using social media and does all her communication on it
+
+**Challenges**
+* Financial Strain: Maria’s earns $20,000 a year, which isn't enough for her expenses. She often has to decide between paying different bills.
+* Lack of Legal Knowledge: She has limited understanding of her rights as a tenant and feels intimidated by legal processes.
+* Access to Resources: Maria has limited access to technology and legal assistance due to her financial situation and work hours.
+
+**Goals**
+* Stability: Maria’s primary goal is to provide a stable home for her children. She wants to avoid eviction to maintain access to her children’s school and her local support network.
+* Education: She is keen to learn more about her rights as a tenant and how she can legally contest an unfair eviction.
+* Assistance: Maria needs easy-to-understand, accessible legal advice and support that fits into her limited budget and time constraints.
+
+:::
+
+#### How sophisticated does the automation need to be?
+
+1. Look over the [Legal app maturity model](https://suffolklitlab.org/legal-tech-class/docs/legal-tech-overview/maturity-model). What level form 
+will be your target for this project? Keep in mind both the sophistication of your user, available time and budget constraints, and any safety concerns.
+
+For a tool aimed at legal professionals, a level 1.5 or level 2 form is usually enough.
+
+For a tool aimed at the general public, a level 2 is usually the minimum. Targeting level 3 or 4 is always tempting, but
+depending on constraints may not be justified by the increased time and cost. You should release a first draft at the earliest
+safe stage, and then continue refining only as long as appropriate based on the needs of the end user for this particular project.
+
+### Identify a project timeline
+
+Come up with a rough schedule with key milestones.
+
+When coming up with a project timeline, think about both resources and the project's
+size.
+
+Resource constraints include:
+
+* Available working hours (a typical developer has 25 productive hours in a week, while a student can allocate about 1/2 of their credit hour expectations to development and may also take 3-5 times as long as an experienced developer at the same tasks)
+* Other ongoing projects
+* Delivering early and frequent work to the client for review and feedback
+
+Complexity is affected by:
+
+* The number of formatted pages (static pages without fields or logic don't add to complexity)
```

Comment:
Not quite right, but it reveals a good reason to clarify this more. Every page of a Word document, if the document has any fields, adds to the overall complexity. A document with no fields or logic at all may as well be one page, since it is unchanging. 

In my experience clients don't really understand the ways that variable length text can cascade the formatting later in the document and they have rarely set up the document in a way to preserve the formatting properly. So you spend time tweaking page 10 if page 2 has the field.